angularjs/angular.d.ts

```diff
@@ -1024,7 +1024,7 @@ declare module ng {
             // Not garanteed to have, since it's a non-mandatory option
             //capacity: number;
         };
-        put(key: string, value?: any): void;
+        put<T>(key: string, value?: T): T;
```

Comment:
oh, sorry. I was read a document. Returns is `the value stored.`. but my comprehension is invalid.